File: lang/python/src/muse/util/TestRunner.py
import sys

TestRunnerItemIndex = 0

def parseTest(ok: bool) -> None:
    global TestRunnerItemIndex

    if ok:
        print("✓ Item [%d] PASSED" % TestRunnerItemIndex)
    else:
        print("✗ Item [%d] FAILED" % TestRunnerItemIndex, file = sys.stderr)

    TestRunnerItemIndex += 1

File: lang/ruby/src/muse/util/TestRunner.rb
module TestRunner

    $TestRunnerItemIndex = 0

    def self.parseTest(ok)
        if ok
            puts(format("✓ Item [%s] PASSED", $TestRunnerItemIndex))
        else
            STDERR.puts(format("✗ Item [%s] FAILED", $TestRunnerItemIndex))
        end

        $TestRunnerItemIndex += 1
    end
end
